relay-server/src/services/buffer/mod.rs

```diff
@@ -483,15 +487,17 @@ mod tests {
     struct EnvelopeBufferServiceResult {
         service: EnvelopeBufferService,
         global_tx: watch::Sender<global_config::Status>,
-        envelopes_rx: mpsc::Receiver<DequeuedEnvelope>,
-        project_cache_rx: mpsc::UnboundedReceiver<ProjectCache>,
+        envelopes_rx: mpsc::Receiver<legacy::DequeuedEnvelope>,
```

Comment:
Should this type actually live in the `legacy` module?

Reply:
Questionable, but for now this is a message which is directed to the "legacy" ProjectCache, that's why I would keep it there for now.